packages/admin/admin/src/stack/breadcrumbs/BreadcrumbOverflow.tsx

```diff
@@ -0,0 +1,42 @@
+import { ChevronRight } from "@comet/admin-icons";
+import { Link, ListItemIcon, ListItemText, Menu, MenuItem } from "@mui/material";
+import { WithStyles } from "@mui/styles";
+import clsx from "clsx";
+import * as React from "react";
+
+import { BreadcrumbItem } from "../Stack";
+import { BreadcrumbLink } from "./BreadcrumbLink";
+import { styles } from "./StackBreadcrumbs.styles";
+
+interface Props {
+    items: BreadcrumbItem[];
+}
+
+// This must return an HTML element, as MuiBreadcrumbs cannot handle fragments.
+export const BreadcrumbOverflow = ({ items, classes }: Props & WithStyles<typeof styles>): React.ReactElement => {
+    const [showOverflowMenu, setShowOverflowMenu] = React.useState<boolean>(false);
+    const overflowLinkRef = React.useRef<HTMLAnchorElement>(null);
+
+    return (
+        <span>
+            <Link
+                ref={overflowLinkRef}
+                onClick={() => setShowOverflowMenu(true)}
+                className={clsx(classes.link, classes.overflowLink)}
+                variant="body2"
+            >
+                ...
```

Comment:
Should we use an icon here?

Reply:
No, according to the [invision design](https://projects.invisionapp.com/d/#/console/21017034/462107630/preview), this should be three dots. Although I did forget the spaces in between, I added them now.